assets/chat/css/onstream.scss

```diff
@@ -52,6 +52,17 @@ i[class^="icon-"],
     letter-spacing: 0.03em;
     .spoiler {
         text-shadow: none;
+        position: relative;
+    }
+    .spoiler:before {
+        content: "[SPOILER]";
+        z-index: 1;
+        font-size: small;
+        color: #828282;
+        font-weight: 500;
+        position: absolute;
+        margin-left: 50%;
+        transform: translate(-50%, 0);
```

Comment:
did you test this with multi-line spoilers?

Reply:
https://i.imgur.com/ZsLS1GI.png it can go off the block at certain widths, but this was the only way i could get to work well